multigeiger/transmission.cpp

```diff
@@ -296,5 +311,59 @@ void transmit_data(String tube_type, int tube_nbr, unsigned int dt, unsigned int
     set_status(STATUS_TTN, ttn_ok ? ST_TTN_IDLE : ST_TTN_ERROR);
     display_status();
   }
+
+}
+
+void transmit_userinfo(String tube_type, int tube_nbr, float tube_factor, unsigned int cpm, unsigned int accu_cpm, float accu_rate,
+                   int have_thp, float temperature, float humidity, float pressure, int wifi_status, bool alarm_status) {
+
+  if (wifi_status != ST_WIFI_CONNECTED)
+    return;
+
+  // if we don't have a valid Messenger config, do not send to Messenger
+  if (sendDataToMessengerEvery < 0)
+    return;
```

Comment:
either remove the empty line 322 or add another empty line after 325 to get consistent?